code/modules/mob/living/say.dm

```diff
@@ -93,6 +93,11 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	if(!message || message == "")
 		return
 
+	var/static/regex/netspeak = regex("\\b(lol|brb|wtf|idk|omg|lmao|btw|ikr|rofl|tbh|nvm|thx|afaik|ooc|ic|pls|plz)\\b", "i")
+	if(!derpspeech && findtext(message, netspeak))
+		to_chat(src, "<span class='warning'>Netspeak is not permitted in character!</span>")
+		return

```

Comment:
should allow you to force it with a warning imo

Reply:
why would we willingly let people say something that can net them a 5 minute ban?

Reply:
There are legitimate cases, like saying "my nephew posted `lol i rekt a scrub` on facebook!", in which you're referring not using

Reply:
also stop using code comments on things that aren't not directly technical feedback